### DISCUSSION TITLE
Show unused "job specification items"

### DISCUSSION THREAD

**Discussion by AdamM-AU** [awsmin/wp-job-openings#31] [Q&A] [answered]
Hi,
We have a Job specification item called "Job Category" We have a few different job types listed in there, but only a few types currently used,only used ones are ever shown to the user, is it possible to show the empty ones as-well in the drop down menu when we simply call [awsmjobs] ? so the public knows we do have other types of job that are advertised from time to time

**Comment by anantajitjg** [comment, marked as answer]
Hi 

Sorry for the delay in response.

By default, only specification options (terms) attached to jobs are only shown in the filters. So to also show these empty terms, you can use the **_awsm_filter_spec_terms_args_** filter hook to modify the default arguments.

```php
function awsm_filter_spec_terms_custom_args( $terms_args ) {
	$terms_args['hide_empty'] = false;
	return $terms_args;
}
add_filter( 'awsm_filter_spec_terms_args', 'awsm_filter_spec_terms_custom_args' );
```
